openshift/ci-operator/update-ci.sh

```diff
@@ -7,31 +7,35 @@ fail() { echo; echo "$*"; exit 1; }
 
 # Deduce branch name and X.Y.Z version.
 BRANCH=$(git rev-parse --abbrev-ref HEAD)
-VERSION=$(echo $BRANCH | sed -E 's/^.*([0-9]+\.[0-9]+\.[0-9]+)|.*/\1/')
+VERSION=$(echo $BRANCH | sed -E 's/^.*(v[0-9]+\.[0-9]+\.[0-9]+|next)|.*/\1/')
 test -n "$VERSION" || fail "'$BRANCH' is not a release branch"
-VER=$(echo $VERSION | sed 's/\./_/;s/\.[0-9]\+$//') # X_Y form of version
+VER=$(echo $VERSION | sed 's/\./_/;s/\.[0-9]\+$//;s/next/nightly/') # X_Y form of version
 
 # Set up variables for important locations in the openshift/release repo.
 OPENSHIFT=$(realpath "$1"); shift
 test -d "$OPENSHIFT/.git" || fail "'$OPENSHIFT' is not a git repo"
-MIRROR="$OPENSHIFT/core-services/image-mirroring/knative/mapping_knative_v${VER}_quay"
+MIRROR="$OPENSHIFT/core-services/image-mirroring/knative/mapping_knative_${VER}_quay"
 CONFIGDIR=$OPENSHIFT/ci-operator/config/openshift/knative-eventing
 test -d "$CONFIGDIR" || fail "'$CONFIGDIR' is not a directory"
 
 # Generate CI config files
-CONFIG=$CONFIGDIR/openshift-knative-eventing-release-v$VERSION
+CONFIG=$CONFIGDIR/openshift-knative-eventing-release-$VERSION
 CURDIR=$(dirname $0)
-$CURDIR/generate-ci-config.sh knative-v$VERSION 4.3 > ${CONFIG}.yaml
-$CURDIR/generate-ci-config.sh knative-v$VERSION 4.4 true > ${CONFIG}__4.4.yaml
-$CURDIR/generate-ci-config.sh knative-v$VERSION 4.5 true > ${CONFIG}__4.5.yaml
+$CURDIR/generate-ci-config.sh knative-$VERSION 4.3 > ${CONFIG}.yaml
+$CURDIR/generate-ci-config.sh knative-$VERSION 4.4 true > ${CONFIG}__4.4.yaml
```

Comment:
I liked your proposal on slack w/ the `__44.yaml` ending, instead of having a `.` in there.

wanna change that ?

Reply:
done -- but this will also change the release branches as well (which could be considered a breaking change)?

Reply:
the release branches ? @lberk how?